packages/components/src/templates/notifications.stories.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import '../solid-components';
+import { html } from 'lit';
+
+export default {
+  title: 'Templates/Notifications',
+  tags: ['!dev'],
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+export const InfoNotification = {
```

Comment:
Set padding-right to 16px 
![Bildschirmfoto 2024-08-29 um 21 30 57](https://github.com/user-attachments/assets/71190abe-2ea2-471d-bf2e-08943ff01947)

Reply:
Updated